service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh

```diff
@@ -310,6 +310,46 @@ fi
 # integration test will ensure that everything in script worked.
 sudo -u "$JUPYTER_USER" sh -c 'terra git clone --all'
 
+################################
+# Setup instance startup service
+################################
+# Create a script to perform the following steps on instance startup:
+# 1. Mount terra workspace resources. This command requires system user home
+#    directories to be mounted. We run the startup service after
+#    jupyter.service to meet this requirement.
+
+# Create the startup script
+readonly TERRA_SETUP_SCRIPT="${USER_TERRA_CONFIG_DIR}/setup-environment.sh"
```

Comment:
i feel like this naming doesn't really explain the purpose.

not sure i have a better suggestion though.  maybe something like "instance-boot.sh", "terra-instance-boot.service"?